docs/development/extensions-core/kinesis-ingestion.md

```diff
@@ -311,6 +311,7 @@ The `tuningConfig` is optional. If no `tuningConfig` is specified, default param
 |`repartitionTransitionDuration`|ISO8601 Period|When shards are split or merged, the supervisor will recompute shard -> task group mappings, and signal any running tasks created under the old mappings to stop early at (current time + `repartitionTransitionDuration`). Stopping the tasks early allows Druid to begin reading from the new shards more quickly. The repartition transition wait time controlled by this property gives the stream additional time to write records to the new shards after the split/merge, which helps avoid the issues with empty shard handling described at https://github.com/apache/druid/issues/7600.|no, (default == PT2M)|
 |`offsetFetchPeriod`|ISO8601 Period|How often the supervisor queries Kinesis and the indexing tasks to fetch current offsets and calculate lag. If the user-specified value is below the minimum value (`PT5S`), the supervisor ignores the value and uses the minimum value instead.|no (default == PT30S, min == PT5S)|
 |`useListShards`|Boolean|Indicates if `listShards` API of AWS Kinesis SDK can be used to prevent `LimitExceededException` during ingestion. Please note that the necessary `IAM` permissions must be set for this to work.|no (default == false)|
+|`skipIgnorableShards`|Boolean|The Supervisor skips closed and empty shards as these need not be processed for ingestion. May help during resharding of stream.|no (default == false)|
```

Comment:
As someone unfamiliar with this piece, I think this documentation can be improved. 
It's not clear, what the recommendation is from the druid side. 

Is there a reason why it's not defaulted to true?

Reply:
The supervisor skips closed and empty shards from consideration for ingestion as these do not have any records.
Since kinesis ingestion with empty shards is not advised, this flag may help during resharding when closed and empty shards may be potentially created.

Reply:
@cryptoe thanks for the feedback. Does the above documentation seem better?
Also, the default value is false as using this may also lead to more AWS calls and potentially increase processing time

Reply:
https://github.com/apache/druid/pull/12792 attempts to add a more robust system where empty shards can be processed by druid. This flag could then be used to ensure that there aren't any tasks processing too many closed and empty shards and avoid uneven task loads

Reply:
We should also mention: Setting this value to true may also lead to more AWS calls and potentially increase processing time.